trace/trace.go

```diff
@@ -105,21 +135,21 @@ func (t *Trace) time() time.Time {
 	return t.startTime // if the trace is incomplete, don't assume an end time
 }
 
-func (t *Trace) writeItem(b *bytes.Buffer, formatter string, startTime time.Time, stepThreshold *time.Duration) {
+func (t *Trace) writeItem(traceNum int32, startTime time.Time, stepThreshold *time.Duration) {
 	if t.durationIsWithinThreshold() {
-		b.WriteString(fmt.Sprintf("%v[", formatter))
-		writeTraceItemSummary(b, t.name, t.TotalTime(), t.startTime, t.fields)
+		totalTime := t.endTime.Sub(t.startTime)
+		klog.InfoS("Trace nested start", logNestArgs(traceNum, t, totalTime)...)
```

Comment:
```suggestion
		klog.InfoS("Trace nested", logNestArgs(traceNum, t, totalTime)...)
```
Let's keep same convention for message name
```
Trace
  Trace nested
  Trace nested end
Trace end
```